app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -134,12 +138,16 @@ public void onCreate(final Bundle savedInstanceState) {
         }
     }
 
+    public boolean isViewBinary(){
+        return !(getActivity() instanceof MainActivity);
```

Comment:
Yes this will move around and have a proper detection, it's just for development currently. And it will only be allowed to show the view mode for such files. Editor text stays empty, as well read/write/saveText disabled.

Reply:
Similar to encryption hook (that reads/writes files differently than normal text) there is another one now that just returns empty string, or returns true when wanting to write. (=dummy). No relevant other change at fragment or someting thus.